packages/dev/addons/src/atmosphere/atmosphere.ts

```diff
@@ -759,6 +759,8 @@ export class Atmosphere implements IDisposable {
         this._physicalProperties.peakOzoneAbsorption = this._peakOzoneAbsorptionKm;
     }
 
+    public uniqueId: number;

```

Comment:
Readonly and use `UniqueIdGenerator` directly (rather than depending on `scene`)
```suggestion
    public readonly uniqueId = UniqueIdGenerator.UniqueId;
```